File: ts/iq/passive~set~passive.ts
/** <iq type="set" xmlns="passive">. Replies with one of: Success. */
export function buildPassiveModePassiveIQ(): BinaryNode {
  return { tag: 'iq', attrs: { type: 'set', xmlns: 'passive', to: 's.whatsapp.net' }, content: [
      { tag: 'passive', attrs: {  } },
    ] }
}

File: rust/iq/passive~set~passive.rs
/// <iq type="set" xmlns="passive">. Replies with one of: Success.
pub fn build_passive_mode_passive_iq() -> Node {
    NodeBuilder::new("iq")
        .attr("type", "set")
        .attr("xmlns", "passive")
        .attr("to", "s.whatsapp.net")
        .child(NodeBuilder::new("passive").build())
        .build()
}